src/pages/index.astro

```diff
@@ -1,55 +1,110 @@
-        </div>
-        <Features />
+    <p>
+      We're currently in the early stages of development and are looking for
+      people to help us test our app. If you're interested in helping us test
+      our app, please sign up for our mailing list below. (We promise not to
+      send you any spam, only important milestones which will result in less
+      than 4-5 emails a year)
+    </p>
+  </section>
+  <section>
+    <form
+      action="https://postal.hackclub.com/subscribe"
+      method="POST"
+      accept-charset="utf-8"
+    >
+      <label for="name">Name</label><br />
+      <input
+        type="text"
+        name="name"
+        id="name"
+        placeholder="R. M. Bubbler"
+        required
+      />
+      <br />
+      <label for="email">Email</label><br />
+      <input
+        type="email"
+        name="email"
+        id="email"
+        placeholder="bubbler@purplebubble.org"
+        required
+      /><br /><br />
+
+      <span>
+        <input type="checkbox" name="gdpr" id="gdpr" required />
+        <strong>Marketing permission</strong>: I give my consent to Purple
+        Bubble to be in touch with me via email using the information I have
+        provided in this form for the purpose of news, updates and marketing.
+        (Required by GDPR)</span
+      >
+      <br /><br />
+      <span
+        ><strong>What to expect</strong>: If you wish to withdraw your consent
+        and stop hearing from us, simply click the unsubscribe link at the
+        bottom of every email we send or contact us at <a
+          href="mailto:%74e%61m%40pu%72p%6Ceb%75bble.org"
+          >t&#101;am&#64;purplebub&#98;le&#46;org</a
+        >.</span
+      >
+      <br /><br /><div style="display:none;">
+        <label for="hp">HP</label><br />
+        <input type="text" name="hp" id="hp" />
       </div>
-    </div>
-  </main>
-  <EmailForm />
-  <Notification />
-  <Footer />
+      <input type="hidden" name="list" value="xjAecLBfJ3HdnmuBXhvkfg" />
+      <input type="hidden" name="subform" value="yes" />
+      <button id="subscribe" type="submit">Subscribe</button>
+    </form>
+  </section>
+  <section>
+    <p>
+      If you would like to donate to Purple Bubble, then you can <a
+        href="https://hcb.hackclub.com/donations/start/purplebubble"
+        >donate through HCB.</a
+      > All donations will go towards the development of the app and are greatly
+      appreciated and tax deductible.
+    </p>
+    <p>
+      You may contact the Purple Bubble team in a couple ways:
+      <ul>
+        <li>
+          Email us at <a
+            href="mailto:
+          %74e%61m%40pu%72p%6Ceb%75bble.org"
+            >t&#101;am&#64;purplebub&#98;le&#46;org</a
+          >
+        </li>
+        <li>
+          Highschoolers only: join the <a href="https://hackclub.com/slack/"
+            >Hack Club Slack</a
+          > and join the #purplebubble channel
```

Comment:
Personally I feel we should remove the bit about the slack, and just encourage anyone interested in helping out with development email me. (or the team email)

Reply:
Email should be the in point, and we can direct/invite to the slack from there

Reply:
that feels more controlled.

Reply:
open to thoughts from @thepurplebubble/innerbubble on this tho.

Reply:
I agree with jasper, lets not recommend slack

Reply:
```suggestion
      You may contact the Purple Bubble team in a couple ways:
      <ul>
        <li>
          Email us at <a
            href="mailto:
          %74e%61m%40pu%72p%6Ceb%75bble.org"
            >t&#101;am&#64;purplebub&#98;le&#46;org</a
          >
        </li>
```

Reply:
that makes sense!

Reply:
done!